app/views/admin/users/_user.html.erb

```diff
@@ -23,6 +23,9 @@
       <ul class="dropdown-menu dropdown-menu-right" role="menu">
         <li><%= link_to t(:discussions, default: 'Discussions'), admin_user_path(user), remote: true %></li>
         <li><%= link_to t(:edit_user, default: 'Edit User'), admin_user_path(user, mode: 'edit'), remote: true %></li>
+        <li><%= link_to t(:delete_user, default: 'Delete User'), admin_user_path(user), remote: true,
+                        method: :delete,
+                        data: { confirm: "Are you sure you want to delete #{user.name}?" } if current_user.is_admin? %></li>
```

Comment:
Yeah I don't think nullify is a great choice here either.  Looking forward to the commit @tomatoturnip!  Thanks for your contributions!